ogusa/tests/test_parameters.py

```diff
@@ -26,12 +26,36 @@ def test_create_specs_object():
     assert specs
 
 
-def test_read_json_params_objects(revision_file):
-    exp = {"revision": {"frisch": 0.3}}
-    act1 = Specifications.read_json_param_objects(JSON_REVISION_FILE)
-    assert exp == act1
-    act2 = Specifications.read_json_param_objects(JSON_REVISION_FILE)
-    assert exp == act2
+# def test_read_json_revision(revision_file):
+#     exp = {"frisch": 0.3}
+#     act1 = Specifications.read_json_revision(JSON_REVISION_FILE)
+#     assert exp == act1
+#     act2 = Specifications.read_json_revision(JSON_REVISION_FILE)
+#     assert exp == act2
+
+
+def test_update_specifications_with_dict():
+    spec = Specifications()
+    new_spec_dict = {
+        'frisch': 0.3,
+    }
+    spec.update_specifications(new_spec_dict)
+    assert spec.frisch == 0.3
+    assert len(spec.errors) == 0
+
+
+def test_update_specification_with_json():
+    spec = Specifications()
+    new_spec_json = """
```

Comment:
@hdoupe This test is not passing because the JSON is not specified properly:
```
if raise_errors and self._errors:
>           raise self.validation_error
E           paramtools.exceptions.ValidationError: {
E               "frisch": [
E                   "Not a valid number: {'value': [{'value': 0.3}]}."
E               ]
E           }

../../ParamTools/paramtools/parameters.py:244: ValidationError
```

What is the proper format to use here?

Reply:
This should work:

```json
    new_spec_json = """
    {
        "frisch": [{"value": 0.3}]
    }
    """
```

Since you're not using labels, you can simplify it to this:

    new_spec_json = """
    {
        "frisch": 0.3
    }
    """

Reply:
Got it - thanks @hdoupe.